### ISSUE TITLE
Possibly add location picture was taken somewhere

### ISSUE THREAD

**Issue by kristen** [jamietsao/codepath-mapstagram#126]
@jamietsao Do you know if the name of the location of the picture is saved in the media object? I was thinking it might be cool to have another tableViewCell that would say where the picture was taken if a location was tagged in instagram. Not sure how it would look, but just a thought.

**Comment by jamietsao** [comment]
Yeah the location name is returned along with the coordinates. I believe it's already being parsed. Check the Media model. Keep in mind the location name isn't always available so we'll need to properly handle that. 

Instagram displays locations as a link where you can see all photos taken at  location. Might be worth adding in the future.
